modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -71,6 +62,49 @@ def validate
 
         private
 
+        def intent_to_file_options(type)
+          options = {
+            intent_to_file_type_code: type,
+            participant_vet_id: params[:participant_vet_id] || target_veteran.participant_id,
+            received_date: Time.zone.now.strftime('%Y-%m-%dT%H:%M:%S%:z'),
+            submitter_application_icn_type_code: ClaimsApi::IntentToFile::SUBMITTER_CODE,
+            ssn: target_veteran.ssn
+          }
+          handle_claimant_fields(options: options, params: params, target_veteran: target_veteran)
+        end
+
+        # BGS requires at least 1 of 'participant_claimant_id' or 'claimant_ssn'
```

Comment:
Note that I brought this over from the v1 controller because it had additional handling for claimant_ssn which was not present in v2 and is used by BGS in cases where the participant_claimant_id is not present.